node/status_node_services.go

```diff
@@ -434,7 +434,7 @@ func (b *StatusNode) NewsFeedService() *newsfeed.Service {
 			nil,
 		)
 
-		if wakuext := b.WakuV2ExtService(); wakuext != nil {
+		if wakuext := b.WakuV2ExtService(); wakuext != nil && wakuext.Messenger() != nil {
```

Comment:
Calling this too early meant that `Messenger()` returned nil and then `NewNewsFeedActivityCenterAdapter(wakuext.Messenger())` wasn't actually `nil`